litestar/channels/backends/redis.py

```diff
@@ -23,6 +23,28 @@
 _XADD_EXPIRE_SCRIPT = (_resource_path / "_redis_xadd_expire.lua").read_text()
 
 
+class _LazyEvent:
```

Comment:
What problem does this solve?

Reply:
It ensures that the `Event` is created within a running event loop. If it’s not, you’ll sometimes get issues where a future within the event itself is attached to a different loop.

This happens in our tests and could also happen when you create an instance of the backend outside an event loop in your application.

Reply:
thanks

Reply:
I should probably put this in the docstring somewhere for future reference